printer_mods/hbo/Voron0_Skirts/Readme.md

```diff
@@ -0,0 +1,86 @@
+# Collection of Voron 0 Skirts
+
+
+
+
+This is a Collection of Voron 0 skirts that provide some additional features and bring the asthetics closer to the bigger VORON brothers.
+
+![Printed Panel](Images/VORON_V0.1_2022.png)
+
+Right now there are 3 variations for the and one commmon rear.
```

Comment:
"3 variations for the...." I think you missed a word here.

Reply:
fixed